quarkus/service/src/intTest/java/org/apache/polaris/service/quarkus/it/QuarkusRestCatalogIT.java

```diff
@@ -0,0 +1,25 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.polaris.service.quarkus.it;
+
+import io.quarkus.test.junit.QuarkusIntegrationTest;
+import org.apache.polaris.service.it.test.PolarisRestCatalogIntegrationTest;
+
+@QuarkusIntegrationTest
+public class QuarkusRestCatalogIT extends PolarisRestCatalogIntegrationTest {}
```

Comment:
Suggestion: if tests can work with the same server, it might be an option to rung them via a JUnit5 "suite". This way we can have only one class that references the _package_ of tests from `polaris-test` and also leverage JUnit5 parallelism as opposed to Gradle test parallelism.

I did not do that under DW only because it was hard to share DW servers between test threads. Now, it looks like all tests are segregated by in entity names and servers are in a separate JVM... so it might work :)

Reply:
Hmm I haven't used JUnit5 suites. Mind if we do your suggestion in another PR?

Reply:
absolutely (it's one additional dependency for the "suite" engine, by the way)